torchagents/utilities/NNBuilder.py

```diff
@@ -0,0 +1,30 @@
+import torch
+from typing import List
+
+def LinearSequentialBuilder(
+        input_size: int,
+        output_size: int,
+        hidden_layers: List[int],
+        output_function: torch.nn = None,
+):
+    activation_function = torch.nn.ReLU
+    net_layers = []
+
+    # Append input layers
+    first_hidden_layer_size = hidden_layers.pop(0)
```

Comment:
Making net_layers looks like it could be a single line list comprehension.

Reply:
Hmm, looking again since we need to keep track of the pairs

Reply:
Yes, it should be feasible to turn this into a list comprehension. This won't make a different in this small test case but for others it should be faster since append is an O(n) operation.

Reply:
Sure. That might require having to add @input_size and @output_size to the @layers list.
Although, take into account that this will ONLY add overhead and not affect the RL loop runtime.